src/formatters/docStringFormatter.ts

```diff
@@ -0,0 +1,9 @@
+import { DocString } from "gherkin-ast";
+import { FormatOptions } from "../index";
+import { lines } from "../utils";
+
+export function format(docString: DocString, options: Partial<FormatOptions>): string {
+    const l = lines(options);
+    l.add('"""', docString.content, '"""');
```

Comment:
I'd create a constant for the delimiters, e.g. `export const DOC_STRING_DELIMITER = "\"\"\"";`